ISISNeutronMuon/MDANSE
# Trajectory loading after conversion

**Description of work**
Following the request for this feature, a checkbox has been added which will make the GUI automatically load the results of a job upon completion.

closes #252 

**Fixes**
1. Added a checkbox to the Action widget.
2. Connected the job output to the Trajectory and Plot Selection tabs using the JobEntry class and JobHolder model.
3. Removed BackEnd.py and ConvertWizard.py, since they are no longer needed.

**To test**
Run a trajectory converter with the autoload box (next to the 'RUN!' button) checked and then unchecked. Check in the Trajectory tab if the trajectory has been loaded.
Run an analysis with the autoload box checked and then unchecked. In each case check if the Plot Creator tab loaded the output file.
In each case, the GUI is expected to load the results when the box is checked, and do nothing after the job has been completed if the box was not checked.
